src/config.js

```diff
@@ -1,17 +1,88 @@
 const {
```

Comment:
Set all new env vars and default values if `undefined` in config.  This will make it much easier to understand what is required in the future.

Reply:
👍